templates/elasticsearch.default.j2

```diff
@@ -63,3 +63,6 @@ export ES_USE_GC_LOGGING=true
 {% if elasticsearch_java_home is defined %}
 JAVA_HOME={{ elasticsearch_java_home }}
 {% endif %}
+{% if ansible_distribution_release == "bionic" %}
+PATH=/usr/lib/jvm/java-1.8.0-openjdk-amd64/bin/:/usr/local/lib/npm/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin
```

Comment:
Hi Santi, I tested with `openjdk-11-jre-headless` and `openjdk-8-jre-headless` both installed and I have not found any problem when using `JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64`

elasticsearch vars used when deploying:
```
# elasticsearch role

elasticsearch_version: "1.7.6"
elasticsearch_apt_java_package: "openjdk-8-jre-headless"
elasticsearch_java_home: "/usr/lib/jvm/java-1.8.0-openjdk-amd64"
elasticsearch_heap_size: "1g"
elasticsearch_max_open_files: "65535"
elasticsearch_timezone: "UTC"
elasticsearch_node_max_local_storage_nodes: "1"
elasticsearch_index_mapper_dynamic: "true"
elasticsearch_memory_bootstrap_mlockall: "true"
elasticsearch_install_java: "true"
elasticsearch_thread_pools:
  - "threadpool.bulk.type: fixed"
  - "threadpool.bulk.size: 50"
  - "threadpool.bulk.queue_size: 1000"
elasticsearch_network_host: "127.0.0.1"
```